scripts/autogen.py

```diff
@@ -42,7 +42,7 @@
     "keras": "https://github.com/keras-team/keras/tree/v2.12.0/",
     "keras_tuner": "https://github.com/keras-team/keras-tuner/tree/v1.3.3/",
     "keras_cv": "https://github.com/keras-team/keras-cv/tree/v0.4.2/",
-    "keras_nlp": "https://github.com/keras-team/keras-nlp/tree/v0.4.1/",
+    "keras_nlp": "https://github.com/keras-team/keras-nlp/tree/r0.5/",
```

Comment:
@fchollet `namex` appears to be breaking all source links when rendering this, because it is trying to map to a path with `src` in it. What's a good fix?

Reply:
Hardcoding a strip of `src` for `keras_nlp` (and soon `keras` as well) source code links...

Reply:
Fix applied.